### DISCUSSION TITLE
Readings in prep for week 14 (after Thanksgiving)

### DISCUSSION THREAD

**Discussion by benmiller314** [benmiller314/dsam2023fall#15] [Reading responses etc]
After reading Onuoha and Mother Cyborg's [People's Guide to AI](https://alliedmedia.org/resources/peoples-guide-to-ai), Crawford and Paglen's [excavation of ML training sets](https://excavating.ai/), the [intro to TextGenEd](https://doi.org/10.37514/TWR-J.2023.1.1.02) and selected exercises from that collection, and ["Where We Are" regarding LLMs](https://compstudiesjournal.com/current-issue-spring-2023-51-1/#:~:text=where%20we%20are) according to various contributors to _Composition Studies_ (especially [Annette Vee](https://compositionstudiesjournal.files.wordpress.com/2023/06/vee.pdf)), **please write a short response to let us know where your mind went**. For example, you might...

* Highlight new ideas or key terms, especially those that you expect will be useful in thinking about digital projects.
* Highlight sentences, passages, or methods/approaches that struck you as especially well done, perhaps worth imitating.
* Link outward to something a passage reminds you of, explaining where you're leading us.
* Pose questions to your classmates about a passage, especially (but not exclusively) if something is confusing.
* NEW! Summarize one of the optional selections that you found either valuable or discussion-worthy.
* NEW! If you tried any of the thought experiments from the People's Guide that you're pretty happy with, feel free to share that!
* Respond to any of the above – which you can now do directly, using threads.

Where possible, please try to point us to at least one particular passage or chunk from the text you're responding to – quoting if you can, giving a timestamp or page reference either way – as a way to help keep our discussion grounded.

**Comment by SidraNArshad** [comment]
In “An Introduction to Teaching with Text Generation Technologies”, a discussion takes place regarding the possible effects AI will have and is currently having on authorship, as well as “intellectual property and citation conventions” generally. Plagiarism in the classroom is a major concern with the introduction of AI tools like CHATgpt and it calls into question the value-gap in education between preparing students to become skilled employees for the workforce and teaching them critical thinking skills—essentially the long-standing battle between capitalism’s complete overtaking of the classroom and (hopefully) instructors’ desires to actually teach their students how to empathetically engage and accurately perceive the world around them. However, there is also the reality of many students becoming desensitized to the education system, as obtaining the best grade by any means is prioritized over learning; thus, students are also more concerned with gaining the skills they’ll need to get ahead in the workforce over learning how to critically write and actually needing that outside of a school-setting. Among the questions asked in the article that pertain to these concerns, one of the most relevant to me is “Will higher education be able to discipline AI to bring it into alignment with academic economies of authorship?”. Marc Watkins offers one possible avenue towards engaging this question in the classroom through his “[AI in First Year Writing Courses](https://wac.colostate.edu/repository/collections/textgened/ethical-considerations/ai-in-first-year-writing-courses/).” Watkins presents assignments in which he asks students to utilize AI-powered assistants to conduct research and conceive of potential counterarguments for their focuses. The assignment distinguishes between what is considered acceptable and unacceptable in terms of AI-assisted writing and writing wholly-composed by AI. After engaging with these AI-tools, Watkins also asks his students to reflect on their experiences with AI—providing them the opportunity to critically examine the affordances and limits of the tools. Watkins's assignment is one example in which students can learn to engage with AI responsibly and ethically, while pushing them to thoughtfully consider what AI is actually capable of and what they are capable of as well.

**Reply by Khushboobhutani** [reply]
While McKee and Porter (2020) rightly point out that AI writing hides or ignores the social and rhetorical contexts of writing to favor an information-transmission model of communication, many writing contexts are satisfied well enough with this stripped-down model of communication. Will a potential misalignment between writing inside and outside of higher ed further devalue the writing for critical inquiry that we assign and practice? No author writes alone, and technological tools have always been part of the entangled materialities that shape writing (Baron). The extent to which AI is constitutive to writing—or acknowledged as such—will depend on disciplinary conventions, individual writing processes, and specific implementations of the technology (Laquintano).

**Comment by Khushboobhutani** [comment]
Annette Vee in her article,[ Large Language Models Write Answers](https://compositionstudiesjournal.files.wordpress.com/2023/06/vee.pdf), contends that writing leads to a productive uncertainty and allows one to wrestle with difficult ideas. This has been the connecting link for me across most of the readings assigned this week. 

Whether it's the large dataset discussed by Tim Laquintano or the massive visual repository of Imagenet, the mammoth data is limited in terms of its worldviews and representations, a concern that we have been engaging with since the beginning of this semester. Alfred L. Owusu Ansah in [Defining Moments, Definitive Programs, and the Continues Erasure of Missing People](https://compositionstudiesjournal.files.wordpress.com/2023/06/owusu-ansah.pdf) underscores this point efficiently as he defines himself as someone who has been historically defined and not a definer. As he grapples with the overwhelmingly alienating representations, he questions if one needs to silence the machines completely or pause them occasionally to think and reflect where the bias is emanating from. This dilemma of working efficiently with AI models is perhaps best illustrated by Anis Bawarshi and Mary Jo Reiff in [Collaborative Writing, Collage, and Cooking: From Humanist to Post-Humanist Assemblages](https://compositionstudiesjournal.files.wordpress.com/2022/07/bawarshi-reiff.pdf) where the authors discuss the distribution of agency across human and non human actors. This has perhaps been one of the most distinct and fun analogies to understand collaborative writing where through the act of cooking, the authors have argued how one can place their thoughts in authoritative dialogue with the voice of others, especially published writers. 

In addition to this, I was also wondering about Crawford and Paglen's article on Excavation of ML training sets where they discuss how the automated interpretation of images reveals deep seated socio-political bias and several unquestioned assumptions that inform the workings of AI. This further led me to think about deepfake images that can challenge normative structures for they might provide the space where the unstable relationship between images and meanings can be utilized. Eg Recent [images ](https://www.forbes.com/sites/mattnovak/2023/11/08/viral-photo-of-benjamin-netanyahu-hospitalized-is-actually-fake/?sh=fa04d2f62b81)of Israeli PM Benjamin Netanyahu lying on a hospital bed in critical condition, or Donald Trump and Putin being [arrested ](https://www.pbs.org/newshour/politics/fake-ai-images-of-putin-trump-being-arrested-spread-online)etc.

**Comment by amj169** [comment]
I feel like the questions “How open is openAI?” and “How open is open access?” (much like Data Feminism prompts us to as data and representation for whom?) get at one huge piece of the AI conversation that is often overlooked (at least by those who I’ve exchanged sentiments about AI and writing with). For example, I find that a lot of people want to focus on what AI can do instead of how it does it, and this reminds me of a generative AI workshop I had the opportunity to attend here at Pitt a few weeks ago. In fact, Annette Vee was one of the panelists who presented her work, but we can talk about that in class because this post addresses a question posed about authorship a couple weeks ago. Authorship was a popular subject in one of the generative AI workshops and the conversation surrounding it was prompted by Robin Kear (and her coresearcher whose name I can’t find or remember right now) who looked into the most popular peer reviewed journals and if they had a policy on the use of ChatGPT and if so, what those policies looked like. They found that basically nobody had any policy that encouraged the use of ChatGPT, but rather, they banned it completely. This sparked conversation about what might happen if/when journals changed their position on this. Perhaps someone might be interested in using ChatGPT to help them rework their methods section to avoid self-plagiarism. Other questions arose about how ChatGPT might be helpful for ESL students who might need help with grammar and tone when writing an article or sending an application out-arguing that these aren’t really altering the writing itself, but rather helping to provide a leg up in a system that prioritizes SAE. Others were quick to point out that this would be problematic because it only reinforces the idea that good English grammar = good writing and that = intelligence.  People also asked about the use of AI powered tools to replace the peer-review process in publishing. How would that affect the field? Some people even wondered if there might eventually be an “academic journal LLM” where everything is published and we worry about peer review later, or not at all (might be worth noting this followed a comment about the speed in which papers might be published if we were allowed to use ChatGPT to write papers). They were thinking that journals couldn’t keep up with the number of submissions, so they would just replace readers with AI or decide not to review it at all and leave it up to the public to digest it as wished. All of these authorship-esque related questions point us to one common thing, and that is that while we will certainly have to adapt to certain changes because of AI, it is important to remember that AI isn’t necessarily the thing that is causing these inequities, but rather, it’s a tool that has allowed us to see the ones that were already present. And it’s what we choose to do and how we choose to handle the ways in which we adapt that are perhaps most important. I think that each of the readings for this week have showed us just that. It is maybe more important than ever during this techno-cultural shift that we worry about how pictures (people!!) are being labeled and standardized (Crawford, Kate, and Paglen), how LLMs are being trained and who is benefiting from white-dominated training models (Laquintano, Schnitzler, Vee) and how we will continue to argue that humans are necessary in things like teaching people processes like those that are essential in learning how to write through inquiry and critical thought (Vee).

**Comment by chloe-cmu** [comment]
Crawford and Paglen's piece was very good - insightful and interesting, and raised important points. It also kind of made my teeth itch, because the piece's historical analysis is very weak. Of course, Crawford and Paglen are not trying to offer a historical analysis, so that might not be a big deal to them. But it's also a good reminder that while reaching across disciplines can be really fruitful for different types of analysis, it can also lead to rookie mistakes. For example, Crawford and Paglen tie labeled images to "echoes of nineteenth century phrenology," but don't actually connect that specific phenomenon and AI image labeling. They also incorrectly historicize Magritte and The Treachery of Images (1929). It's more appropriate to situate Magritte in the interwar period, when artists used methods like surrealism and dadaism to explore the chaos and disillusionment of the early 1900s. It's a little ironic, because Crawford and Paglen are right to insist that the specifics of how AI labelling happens is important to understand, but then mischaracterize the specifics of other areas as a rhetorical hook.  

This week's readings were also interesting in that they prompted me to think about teaching with generative AI. I definitely understand the urge to ban AI outright -- I don't have a class I run yet, but it seems like an overwhelming problem to grapple with. But we can't actually stop students from using AI. And, these tools exist in the world, and students need to know how to navigate with and around them. I really enjoyed reading about various assignments that instructors have come up with to support students in using AI, in particular Paul Fyre's Paper Turing Test because of its focus on what does or does not count as cheating, and kathy wu's Who's Talking, because of its emphasis on the question of authorship in found text. Both of these assignments seem well-suited to helping a student think about their position in the writing process, the AI's position, and how they might bridge the gap.

**Comment by Rezaetmd** [comment]
Here are my thoughts about the two texts that I perused:
In the first text, Excavating AI, we understand that there are important societal and political implications associated with employing photos to train AI systems. Crawford and Paglen (2020) argue that automated picture interpretation is not only a technical endeavor but also fundamentally social and political. They stress how AI systems are being incorporated more and more into societal institutions, such as selecting candidates for jobs, determining whether students pay attention in class, and identifying suspicious individuals. Given that AI systems are based on datasets that represent underlying social and political norms and presumptions, the authors argue that it is imperative to comprehend the politics within these systems. They explain further that large-scale labeled picture training for AI systems is a political intervention in and of itself. Images can be labeled in a way that reflects and reinforces prejudices and stereotypes, which affects how AI systems categorize people according to factors like ethnicity, gender, emotions, ability, sexual orientation, and personality. This prompts questions concerning the effects of applying these labels to social judgments and decision-making.

The second text, TextGenEd, explains how the use of large language models, such as GPT-3, in writing instruction presents a number of potential ethical issues. Since these models have been regarded as student cheating tools, there is cause for concern regarding the social stigma attached to them, which could influence how widely accepted and utilized they become [1]. Furthermore, because these models mostly use "standard" grammar, there is a possibility of style and language bias, which could have an impact on how people perceive "accent" in writing and language diversity. Regarding the standard grammar that is used in these writings, there have been a raise in the use of specific softwares that assumingely can detect a text that is written by a student (or any human being) and one that is generated by these language tools. I'm wondering here how their use might shape the way students write their academic articles.

**Comment by MustafaKandil** [comment]
In this week’s readings, the main point is that how people who is a student, author, academician, politician etc. look like to AI and how to use it. First of all, Onuoha and Mother Cyborg's [People's Guide to AI](https://alliedmedia.org/resources/peoples-guide-to-ai), Crawford and Paglen's [excavation of ML training sets](https://excavating.ai/) talk about what is the AI and how it works. Especially, I liked People’s Guide to AI because of its framework giving us what AI is and woking logic basically. Also, both of readings talk about equity and fairness,justice but as Crawford and Paglen mentioned there is no equity and neutral, objectiveness of using technology. They said “Datasets aren’t simply raw materials to feed algorithms but are political interventions. As such, much of the discussion around “bias” in AI systems misses the mark: there is no “neutral,” “natural,” or “apolitical” vantage point that training data can be built upon.” Also, Onucha and Cyborgh said “Because we live in a society that is built on top of structural racism and inequity, these technologies don't always end up being good for everyone” AI and other technology might be a great tools that can allow us to better to the things we want to do as they mentioned but also it might be most-evil weapon for whom want to end what they want up, want to control and manage politic, economy, power and humans life. An example came in my mind, it was how facebook use peoples data to manipulate to election! Onucha and Cyborgh alos emphasized “The companies creating predictive policing algorithms are doing so in order to make money and sell the software to police departments. But the people who will be affected by the algorithms (such as the people who live in the areas that are classified as high-crime or the people who might be affected as a result of increased police presence in their neighborhoods) don’t have a say in these algorithms. Even worse, many of these people don’t know that their data has been collected or that these algorithms are being used in the first place. These people will not even have the opportunity to access the data or algorithms and determine for themselves whether or not they are fair.”  These made me think that it is really efficient tools that help us or just for them holding power! Can we really use it for everyone and in a good way for humanities? 
Also, as an international student who doesn’t know how to use the second language effectively, AI and tools help me a lot to do my research and studies in here and pursue my education easily, doing and learning better, these are like very talented assistant but sometimes I feel guilty while I use tools because I have been raised with traditional perspective of education and also I love and agree with what Vee said “Let’s remind ourselves—and our students— how writing can help us learn and can help us develop new questions. How it can help us to heal trauma, recognize love, and understand new perspectives.”
